packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -31828,12 +31828,6 @@ Map {
   "DashboardIcon" => Object {},
   "DataScientistIcon" => Object {},
   "DataTableIcon" => Object {},
-  "Emptystate404Icon" => Object {},
-  "EmptystateDefaultIcon" => Object {},
-  "EmptystateErrorIcon" => Object {},
-  "EmptystateNoresultsIcon" => Object {},
-  "EmptystateNotauthorizedIcon" => Object {},
-  "EmptystateSuccessIcon" => Object {},
```

Comment:
Shouldn't we re-add the modified versions of these back into the export?

Reply:
yes. good catch.

Reply:
Actually I just did a build and they are there. We should be good. Looking at why they are not present in the snapshot